### PR TITLE
Test code review tools

### PR DESCRIPTION
**Pull request by WhatWouldHoneyBadgerDo**
This change is [Reviewable](https://reviewable.io/reviews/whatwouldhoneybadgerdo/hello-world/3)

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/WhatWouldHoneyBadgerDo/hello-world/pull/3" target="_blank">ReviewNinja

File: SimpleMongoRepository.java [hunk 1]
```diff
@@ -252,4 +252,8 @@ public void deleteAll() {
 	private static int tryDetermineRealSizeOrReturn(Iterable<?> iterable, int defaultSize) {
 		return iterable == null ? 0 : (iterable instanceof Collection) ? ((Collection<?>) iterable).size() : defaultSize;
 	}
+
+	private static double guessSize() {
+		return Math.random() + 1;
```
Comment (WhatWouldHoneyBadgerDo) [inline SimpleMongoRepository.java]: Plus one???

Comment (WhatWouldHoneyBadgerDo) [inline SimpleMongoRepository.java]: !resolved Yep, plus one.

**Comment by WhatWouldHoneyBadgerDo** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[SimpleMongoRepository.java, line 257 at r1](https://reviewable.io:443/reviews/whatwouldhoneybadgerdo/hello-world/3#-KW-9neIODWiS7LlQjGO-r1-257:-KW-BFRdonk-HqRA_SEn:b-896fix) ([raw file](https://github.com/whatwouldhoneybadgerdo/hello-world/blob/724433d5c21238557ad02e25548d531bedb47ac1/SimpleMongoRepository.java#L257)):*
<details><summary><i>Previously, WhatWouldHoneyBadgerDo (Rebecca Drabenstott) wrote…</i></summary>
> !resolved Yep, plus one.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/whatwouldhoneybadgerdo/hello-world/3)*

**Comment by WhatWouldHoneyBadgerDo** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[SimpleMongoRepository.java, line 257 at r1](https://reviewable.io:443/reviews/whatwouldhoneybadgerdo/hello-world/3#-KW-9neIODWiS7LlQjGO-r1-257:-KW-BNYtby5FSF62e6Sw:b-896fix) ([raw file](https://github.com/whatwouldhoneybadgerdo/hello-world/blob/724433d5c21238557ad02e25548d531bedb47ac1/SimpleMongoRepository.java#L257)):*
<details><summary><i>Previously, WhatWouldHoneyBadgerDo (Rebecca Drabenstott) wrote…</i></summary>
> Done.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/whatwouldhoneybadgerdo/hello-world/3)*

**Pull request closed without merging**